src/controllers/token.rs

```diff
@@ -130,6 +130,8 @@ pub async fn new(app: AppState, req: BytesRequest) -> AppResult<Json<Value>> {
             .transpose()
             .map_err(|_err| bad_request("invalid endpoint scope"))?;
 
+        let recipient = user.email(conn)?;
```

Comment:
This performs an extra database query to resolve the user's email address - is this the best way to get it?

Reply:
yeah, that's fine. we could move sending the email to a background job, but that would be yet another query. this is fine as is for now :)